### ISSUE TITLE
Question on scratchpad - may i cross reference linked JS from Scratchpad

### ISSUE THREAD

**Issue by natanraj** [eclipsesource/tabris-js#1026]
Assume I have linked SomeUtil.js already written and do i have option to use the reference the same in scratchpad editor and use it? if yes, what way can I do it?

Currently my modularly written code is copied inside my scratchpad and becomes big as I grow.

**Comment by cookieguru** [comment]
You could:
* Retrieve it over HTTP and then `eval` it
* Store the code on your development machine and use the Tabris.js developer app to dynamically load your code

**Comment by natanraj** [comment]
Let me rephrase my question .. . I have a scratchpad code using three or four modules .. Now i can write them separate as the way I do (saying require('myModule').  Is this similalry possible in the linked scripts (appearing under MyScripts)

**Comment by cookieguru** [comment]
Scratchpad supports `require`.  But there's no way to update the `package.json` to load your own modules nor is there a way to upload arbitrary files.  The scratchpad is mostly built for getting one's feet wet with the platform and testing snippets.
Is there a reason you can't host the code on your development machine?

**Comment by natanraj** [comment]
Thanks for the update. The main reason why i wanted is - to avoid http / eval.
